lib/MusicBrainz/Server/Report/LimitedEditors.pm

```diff
@@ -0,0 +1,33 @@
+package MusicBrainz::Server::Report::LimitedEditors;
+use Moose;
+
+use MusicBrainz::Server::Constants qw( $EDITOR_MODBOT );
+
+with 'MusicBrainz::Server::Report::EventReport';
```

Comment:
This looks unnecessary.

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/556/commits/125982facf0c08a9f6f34d581c15fbd36f513416